apps/mobile/src/navigation/linking/index.js

```diff
@@ -82,23 +83,23 @@ export const routingConfig = {
   '/messages':                                               `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Messages Tab/Messages`,
 
   // catch-alls
-  '(.*)/group/:groupSlug':                                   `${AUTH_ROOT_SCREEN_NAME}/${modalScreenName('Group Explore')}`,
-  '(.*)/group/:groupSlug/explore':                           `${AUTH_ROOT_SCREEN_NAME}/${modalScreenName('Group Explore')}`,
-  '(.*)/members/:id':                                        `${AUTH_ROOT_SCREEN_NAME}/${modalScreenName('Member')}`,
-  '(.*)/post/:id':                                           `${AUTH_ROOT_SCREEN_NAME}/${modalScreenName('Post Details')}`,
-  '(.*)/post/:id/comments/:commentId':                       `${AUTH_ROOT_SCREEN_NAME}/${modalScreenName('Post Details')}`,
-  '(.*)/create/group':                                       `${AUTH_ROOT_SCREEN_NAME}/Create Group`,
-  '(.*)/create/post':                                        `${AUTH_ROOT_SCREEN_NAME}/Edit Post`,
-  '(.*)/post/:id/edit':                                      `${AUTH_ROOT_SCREEN_NAME}/Edit Post`,
+  ':unmatchedBasePath(.*)/group/:groupSlug':                  `${AUTH_ROOT_SCREEN_NAME}/${modalScreenName('Group Explore')}`,
```

Comment:
now names the unmatched part of the path and also sends that along to screen params.

Reply:
oh, so we can feed that into url sharing, error reporting and analytics?